crates/bevy_mesh/Cargo.toml

```
@@ -0,0 +1,37 @@
+[package]
+name = "bevy_mesh"
+version = "0.15.0-dev"
+edition = "2021"
+description = "Provides image types for Bevy Engine"
```

I think this should be changed 😅
```suggestion
description = "Provides mesh types for Bevy Engine"
```

---

lmaooooo good catch